src/devices/cpu/axc51/axc51.cpp

```diff
@@ -121,11 +138,19 @@ device_memory_interface::space_config_vector axc51base_cpu_device::memory_space_
 
 /* Read/Write a byte from/to the Internal RAM indirectly */
 /* (called from indirect addressing)                     */
-uint8_t axc51base_cpu_device::iram_iread(offs_t a) { return m_data.read_byte(a); }
-void axc51base_cpu_device::iram_iwrite(offs_t a, uint8_t d) { m_data.write_byte(a, d); }
+/* these go through DBASE register on axc51 (at least stack accesses) */
+//uint8_t axc51base_cpu_device::iram_indirect_read(offs_t a) { return m_data.read_byte(a); }
+//void axc51base_cpu_device::iram_indirect_write(offs_t a, uint8_t d) { m_data.write_byte(a, d); }
+uint8_t axc51base_cpu_device::iram_indirect_read(offs_t a) { return m_data.read_byte((m_sfr_regs[AXC51_DBASE - 0x80] * 4) + a); }
+void axc51base_cpu_device::iram_indirect_write(offs_t a, uint8_t d) { m_data.write_byte((m_sfr_regs[AXC51_DBASE - 0x80] * 4) + a, d); }
+
+
+
+#define IRAM_INDIRECT_R(a)      iram_indirect_read(a)
+#define IRAM_INDIRECT_W(a, d)   iram_indirect_write(a, d)
+
+
 
-#define IRAM_IR(a)      iram_iread(a)
-#define IRAM_IW(a, d)   iram_iwrite(a, d)
```

Comment:
Does this actually achieve anything besides obfuscation?  You may as well just search/replace them to use the actual member function names.

Reply:
Well I'm trying to stick to the style of the code, which as we know, isn't exactly great.  All I've done here is a search+replace rename because it was annoying me trying to read it.

Reply:
I'd just run a search-and-replace to get rid of the macros.  That *was* the style a long time ago but the macros add no value here.

Reply:
I've replaced a whole bunch of the macros, including this one.